examples/docker-compose-ngrok.yml

```diff
@@ -0,0 +1,24 @@
+version: "3.8"
+
+services:
+  mc:
+    image: itzg/minecraft-server
+    tty: true
+    stdin_open: true
+    ports:
+      - "25565:25565"
+    environment:
+      EULA: "TRUE"
+    restart: unless-stopped
+    volumes:
+      # attach the relative directory 'data' to the container's /data path
+      - ./data:/data
+
+  ngrok:
+    image: shkoliar/ngrok:latest
+    ports:
+      - 4551:4551
+    links:
+      - mc

```

Comment:
Did you happen to try without this? Docker compose network includes the name resolving of the other services by default, so `mc` should have already been resolvable.

Reply:
Id does appear to work without it, however I left it in as that's how the creator of the image suggested usage is setup, I assume its there for redundancy sake. Up to you though if ya wanna remove it.

Reply:
Yeah, it would be great to remove it.